apps/frontend/core-dapp/src/components/Link.tsx

```diff
@@ -1,25 +1,41 @@
 import NextLink from 'next/link'
-import styled from 'styled-components'
+import styled, { css, SimpleInterpolation } from 'styled-components'
 
-const Anchor = styled.a`
-  :hover {
-    color: ${({ theme }): string => theme.color.primary};
-  }
+const Anchor = styled.a<{ noUnderline?: boolean }>`
+  color: ${({ theme }): string => theme.color.primaryLight};
+  ${({ noUnderline }): SimpleInterpolation | string =>
+    noUnderline
+      ? ''
+      : css`
+          text-decoration: underline;
+          :hover {
+            text-decoration: underline;
+          }
+        `}
 `
 
 type Props = {
   href: string
-  target?: string
+  target?: '_self' | '_blank'
   className?: string
   scroll?: boolean
+  noUnderline?: boolean
 }
 
-const Link: React.FC<Props> = ({ className, href, target = '_self', children, scroll }) => (
+const Link: React.FC<Props> = ({
+  className,
+  href,
+  target = '_self',
+  children,
+  noUnderline,
```

Comment:
I personally dislike boolean variable names that contain a negation in their name. For example I tend to replace `isBalanceInsufficient` with `isBalanceSufficient (= !isBalanceInsufficient)` or `isBalanceExceeded`

In this case I'd do:

```suggestion
  underline = true,
```

Since it's a new prop, we don't need to worry about breaking anything.